client/src/styles/fluent.scss

```diff
@@ -1,3 +1,4 @@
+// sass-lint:disable no-important, class-name-format
```

Comment:
[nitpick] consider disabling per line (eg ` // sass-lint:disable-line no-important`) rather than the whole file